Gems/ROS2/Code/Source/RobotImporter/FixURDF/FixURDF.h

```diff
@@ -0,0 +1,33 @@
+/*
+* Copyright (c) Contributors to the Open 3D Engine Project.
+* For complete copyright and license terms please see the LICENSE at the root of this distribution.
+*
+* SPDX-License-Identifier: Apache-2.0 OR MIT
+*
+*/
+
+#pragma once
+
+#include <AzCore/XML/rapidxml.h>
+#include <AzCore/std/string/string.h>
+#include <AzCore/std/containers/vector.h>
+
+
+namespace ROS2
+{
+   namespace Utils
+   {
+       //! Modifies in memory URDF to add missing inertia to links, circle navigate SDF error 19
+       //! @param urdf - the in memory URDF to modify
+       //! @returns a list of links that were modified
+       AZStd::vector<AZStd::string> AddMissingInertiaToLink(AZ::rapidxml::xml_node<>* urdf);
+
+       //! Modifies names of links and joints to be unique, circle navigate SDF error 2
+       //! @param urdf - the in memory URDF to modify
+       //! @returns a list of links that were modified
+       AZStd::vector<AZStd::string> ChangeDuplications(AZ::rapidxml::xml_node<>* urdf);
+
+       std::string ModifyURDFInMemory(const std::string& data);
```

Comment:
I think we should put this option to add missing `<inertial>` tags and rename duplicate links and joints into two new settings that is apart of the [SdfAssetBuilderSettings](https://github.com/o3de/o3de-extras/blob/development/Gems/ROS2/Code/Source/SdfAssetBuilder/SdfAssetBuilderSettings.cpp#L60-L94).

We can source those settings from the [Settings Registry](https://github.com/o3de/o3de-extras/blob/development/Gems/ROS2/Code/Source/SdfAssetBuilder/SdfAssetBuilderSettings.cpp#L106-L114) and they would be available for the user to select via the File Selection Page in the Robot Importer.